### ISSUE TITLE
Hyperparameters for reproduction

### ISSUE THREAD

**Issue by smart0eddie** [lkyddd/GradMA#1]
Hi
May you please shared the exact command to reproduce the results on the paper?
The following is the CIFAR-10 trained using the beta and gamma you provide in the readme.
The final model reaches about 0.45, which is 0.2 lower than the 0.65 reported in the paper.

{"comm_round": 1000, "I": 5, "batch_size": 64, "eval_step_interval": 5, "eval_batch_size": 2048, "eval_on_full_test_data": true, "gamma_l_list": [[0, 0.001], [100, 0.001], [200, 0.001]], "gamma_g_list": [[0, 1.0], [100, 1.0], [200, 1.0]], "beta_1_list": [[0, 0.5], [100, 0.5], [200, 0.5]], "beta_2_list": [[0, 0.5], [100, 0.5], [200, 0.5]], "rho_l": 0.0, "rho_g": 0.0, "weight_decay": 0.0, "model_type": "Lenet_5", "data_set": "CIFAR-10", "data_partition_mode": "non_iid_dirichlet", "non_iid_alpha": 0.1, "client_num": 100, "memory_num": 100, "selected_client_num": 10, "device": "cuda", "seed": 0, "app_name": "GradMA_SW"}
![test_acc](https://github.com/lkyddd/GradMA/assets/4142838/78578ee7-23d6-4dfa-99e0-90bfcf4a9bb5)

Thank you.

**Comment by lkyddd** [comment]
Thank you for your interest in our work. The results reported in our paper are the best accuracy based on a grid search of multiple hyper-parameters. We also provide the parameters of grid search in the paper and label them in the figure below.
In addition, which field do you study federated learning? We can cooperate if there is an opportunity.


------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "lkyddd/GradMA"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年3月27日(星期三) 晚上11:44
***@***.***&gt;;
***@***.***&gt;;
主题:&nbsp;[lkyddd/GradMA] Hyperparameters for reproduction (Issue #1)





 
Hi
 May you please shared the exact command to reproduce the results on the paper?
 The following is the CIFAR-10 trained using the beta and gamma you provide in the readme.
 The final model reaches about 0.45, which is 0.2 lower than the 0.65 reported in the paper.
 
{"comm_round": 1000, "I": 5, "batch_size": 64, "eval_step_interval": 5, "eval_batch_size": 2048, "eval_on_full_test_data": true, "gamma_l_list": [[0, 0.001], [100, 0.001], [200, 0.001]], "gamma_g_list": [[0, 1.0], [100, 1.0], [200, 1.0]], "beta_1_list": [[0, 0.5], [100, 0.5], [200, 0.5]], "beta_2_list": [[0, 0.5], [100, 0.5], [200, 0.5]], "rho_l": 0.0, "rho_g": 0.0, "weight_decay": 0.0, "model_type": "Lenet_5", "data_set": "CIFAR-10", "data_partition_mode": "non_iid_dirichlet", "non_iid_alpha": 0.1, "client_num": 100, "memory_num": 100, "selected_client_num": 10, "device": "cuda", "seed": 0, "app_name": "GradMA_SW"}
 test_acc.jpg (view on web)
 
Thank you.

**Comment by smart0eddie** [comment]
Thank you very much for your response.
We think your paper is nice and looking for the possibility to further extend it.
Therefore we are trying to firstly reproduce the results
